docs/example_scripts/example_submission.py

```diff
@@ -31,8 +31,8 @@
 CODALAB_URL = 'http://localhost/'
 USERNAME = 'admin'
 PASSWORD = 'admin'
-PHASE_ID = None
-TASK_LIST = []
+PHASE_ID = 31
+TASK_LIST = [36, 37]
```

Comment:
these shouldn't come pre-set I don't think? maybe add an assertion somewhere below:

```python
assert PHASE_ID, "Missing PHASE_ID. Please fill in all of the variables at the top of this file."
assert len(TASK_LIST) > 0, "Missing TASK_LIST entries. Please fill in all of the variables at the top of this file."
```

Maybe overkill..

Reply:
@gibsonbailey already did this on line 47, and an empty task list runs the submission on all of the tasks on a phase.